src/AutoRest.CSharp.V3/Generation/Writers/RestClientWriter.cs

```diff
@@ -542,7 +532,16 @@ private void WriteStatusCodeSwitch(CodeWriter writer, CodeWriterDeclaration mess
                         }
                         else
                         {
-                           writer.Line($"case {statusCode.Family}:");
+                            if (statusCode.Family == 2)
+                            {
+                                writer.Line($"case 200:");
```

Comment:
You don't need this line for the family case. The example had it both ways to show that we can mix the individual codes and ranges.

` writer.Line($"case int s when s >= {statusCode.Family*100:L} && s < {statusCode.Family * 100 + 99:L}:");

Reply:
Ahh, yess!